src/core/evaluator.js

```diff
@@ -381,6 +385,43 @@ class PartialEvaluator {
     return data;
   }
 
+  async fetchStandardFontData(name) {
+    // The symbol fonts are not consistent across platforms, always load the
+    // font data for them.
+    if (
+      this.options.useSystemFonts &&
+      name !== "Symbol" &&
+      name !== "ZapfDingbats"
+    ) {
+      return null;
+    }
```

Comment:
> In follow ups:
> use foxit fonts when disableFontFace is true

I believe that you could quite easily support at least *this* part immediately, simply by doing:
```suggestion
    if (!this.options.disableFontFace) {
      // The symbol fonts are not consistent across platforms, always load the
      // font data for them.
      if (
        this.options.useSystemFonts &&
        name !== "Symbol" &&
        name !== "ZapfDingbats"
      ) {
        return null;
      }
    }
```